CONTRIBUTING.md

```diff
@@ -93,15 +93,18 @@ Writing good commit logs is important. A commit log should describe what
 changed and why. Follow these guidelines when writing one:
 
 1. The first line should be 50 characters or less and contain a short
-   description of the change prefixed with the name of the changed
-   subsystem (e.g. "net: add localAddress and localPort to Socket").
+   description of the change. All words in the description must be in lowercase
+   with the exception of the ones that refer to code (e.g. function/variable
+   names). The description must be prefixed with the name of the changed
+   subsystem and should start with a present-tense verb (e.g. "net: add
```

Comment:
@gibfahn I used `present-tense` here as per this comment: https://github.com/nodejs/node/pull/8479#r78279077.

> Not sure but there seems to be a risk of bike-shedding.

The risk is confirmed. I'm happy to use whatever is best for the purpose.